guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -91,7 +95,13 @@ Ensure {SmartProxy} has access to `{RepoRHEL8ServerSatelliteMaintenanceProductVe
 +
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-maintain} self-upgrade
+# subscription-manager repos --enable \
+{RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+
+# subscription-manager repos --enable \
+{RepoRHEL8ServerSatelliteCapsuleProductVersion}
```

Comment:
AFAIK we pull content from Satellite and not directly via the CDN so I think this is invalid. I think the line above this should ensure that.

Reply:
We do, but the repo names will be identical, so this should still work.